README.md

```
@@ -38,25 +33,22 @@ _block
 _endblock
 ```
 
-
-
 Compiling
+
 ---------
 ```bash
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)